src/dbnode/namespace/dynamic_options.go

```diff
@@ -40,11 +40,12 @@ var (
 )
 
 type dynamicOpts struct {
-	iopts                  instrument.Options
-	csClient               client.Client
-	nsRegistryKey          string
-	initTimeout            time.Duration
-	forceColdWritesEnabled bool
+	iopts                              instrument.Options
+	csClient                           client.Client
+	nsRegistryKey                      string
+	initTimeout                        time.Duration
+	forceColdWritesEnabled             bool
+	allowEmptyInitialNamespaceRegistry bool
```

Comment:
Should this default to true in the options?

Reply:
I think we want this to default to false. DB nodes, as of today, wait indefinitely until they receive some namespace information. We probably want to default to what's safest for those guys which is ensuring this is false.

Reply:
Yeah I suppose; figure this was the blessed path going forwards, so anyone who's been running will be fine, as they should have the existing namespaces registered, right? I guess maybe we should flip it to true once the full flow has landed 👍